tic-tac-toe/src/components/board/board.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Square from '../square/square';
+import './board.css';
+
+class Board extends Component {
+    renderSquare(i) {
+        return (
+            <Square
+                value={this.props.squares[i]}
+                onClick={() => {this.props.onClick(i)}}
+                key={i}
+            />
+        );
+    }   
+    render() {
+        const outerClassName = this.props.outerClassName;
```

Comment:
use object destructuring

Reply:
How I am not pass classNames in props , I use style-module.
```suggestion
return (
    <div className={classes.Board}>
      {squareComponents}
    </div> 
  )
```